tox.ini

```diff
@@ -3,11 +3,21 @@
 # test suite on all supported python versions. To use it, "pip install tox"
 # and then run "tox" from this directory.
 
+# Map Python versions in GitHub Actions to tox environments to run.
+[gh-actions]
+python =
+    2.7: py27-coverage,codechecks
+    3.6: py36-coverage
+    3.7: py37-coverage
+    3.8: py38-coverage
+    3.9: py39-coverage,typechecks
+    pypy-3.7: pypy3
```

Comment:
I noticed that pypy3 has been added in tox configuration here, but not in GitHub Actions workflow.  Is that a deliberate omission?

Reply:
Was going to do this in this ticket, but ran into bottlenecks so ended up moving it out into separate ticket.